src/client/css/style.css

```diff
@@ -1703,7 +1703,7 @@ section#contact_form .text-danger {
   font-weight: 700;
   width: 200px;
   height: 50px;
-  margin: 0 0 0 22px;
+  /*margin: 0 0 0 20px;*/
```

Comment:
Commenting this line seems to make it deviate from the center on lg screens. But I would hate for it to work through adjusting margins magic numbers via media queries.